check_cron_or_pr.sh

```diff
@@ -22,9 +22,8 @@ else
 	export SHOULD_DEPLOY="yes"
 fi
 
-if [[ "${GITHUB_ENV}" ]]; then
-  echo "GITHUB_BRANCH=${GITHUB_BRANCH}" >> "${GITHUB_ENV}"
-	echo "SHOULD_BUILD=${SHOULD_BUILD}" >> "${GITHUB_ENV}"
-	echo "SHOULD_DEPLOY=${SHOULD_DEPLOY}" >> "${GITHUB_ENV}"
-  echo "VSCODE_QUALITY=${VSCODE_QUALITY}" >> "${GITHUB_ENV}"
-fi
+[[ "${GITHUB_ENV}" ]] && {
```

Comment:
Can you put the `if` block and the multiple `echo/>>` lines? I find it more readable.